Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -14,7 +14,7 @@ This file defines the type of points on a Weierstrass curve as an inductive, con
 at infinity and affine points satisfying a Weierstrass equation with a nonsingular condition. This
 file also defines the negation and addition operations of the group law for this type, and proves
 that they respect the Weierstrass equation and the nonsingular condition. The fact that they form an
-abelian group is proven in `Mathlib.AlgebraicGeometry.EllipticCurve.Group`.
+abelian group is proven in `Mathlib/AlgebraicGeometry/EllipticCurve/Group.lean`.
```

Comment:
Please note that `Mathlib.AlgebraicGeometry.EllipticCurve.Group` is not working in this case; see https://leanprover-community.github.io/mathlib4_docs/Mathlib/AlgebraicGeometry/EllipticCurve/Affine.html .

> can you do the same for lines 14-15 in `Group.lean`?

Sure.

Reply:
Yeah, I am very aware of this annoying bug in the docs! Do you have any idea why?

Reply:
According to https://github.com/leanprover/doc-gen4/blob/b6ae1cf11e83d972ffa363f9cdc8a2f89aaa24dc/DocGen4/Output/DocString.lean#L68-L100 the only correct way to reference file is write its filename, otherwise I don't expect it works (seems that it works in some cases, but I think it only works if the file referenced is already `import`ed by current file), since there is no symbol named `Mathlib.AlgebraicGeometry.EllipticCurve.Group`...

Reply:
> can you do the same for lines 14-15 in `Group.lean`?

The line break in that file is too tight. 🤦‍♂️ Since it's working, I'll not change them.

Reply:
I think this works?
```
under the geometric group law defined in `Mathlib.AlgebraicGeometry/EllipticCurve/Affine.lean` and
in `Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean`.
```

Reply:
In any case, I guess don't worry too much about it because they will all be refactored eventually (hopefully soon).